raspberrypi-firmware/pkg.yaml

```diff
@@ -6,10 +6,10 @@ dependencies:
 steps:
 # {{ if eq .ARCH "aarch64" }} This in fact is YAML comment, but Go templating instruction is evaluated by bldr restricting build to arm64 only
   - sources:
-      - url: https://github.com/raspberrypi/firmware/archive/1.20210201.tar.gz
+      - url: https://github.com/raspberrypi/firmware/archive/refs/tags/1.20210805.tar.gz
```

Comment:
May you please update the URL to https://github.com/raspberrypi/firmware/archive/refs/tags/1.20210805.tar.gz ? That will help us with future automatic updates (https://github.com/talos-systems/bldr/issues/48).